client/components/formatted-header/style.scss

```diff
@@ -1,4 +1,7 @@
 .formatted-header {
+	display: flex;
```

Comment:
This feels a bit risky to me. When I search for how many places have overriden the `.formatted-header` styles there's [a lot of places](https://github.com/Automattic/wp-calypso/search?l=SCSS&q=.formatted-header)! I'm worried we could break them in unexpected ways if we were to change something as fundamental as the `display` rule. Same goes with introducing the `formatted-header__heading` wrapper.

What about allowing `<FormattedHeader>` to accept a `children` prop that sticks any children after the subheading. [Looks like it's already supposed to accept a `children` prop](https://github.com/Automattic/wp-calypso/blob/15b411f4a8e2f071d608eb86b125fcc4567b1d5c/client/signup/step-wrapper/index.jsx#L129). That way we could put a button into the header, although we'd need to position it in `client/signup/style.scss` using some sort of `position: absolute` shinanigans. Not as nice as using flex, but it means we wouldn't be changing `client/components/formatted-header/style.scss` which means not (potentially) breaking a lot of existing usage.

Reply:
> Looks like it's already supposed to accept a children prop

This one doesn't work because `FormattedHeader` never use it 😅

Reply:
I've changed to wrap the `FormattedHeader` into the `step-wrapper__header` with action buttons. I think this would be the low risky solution. How do you think?